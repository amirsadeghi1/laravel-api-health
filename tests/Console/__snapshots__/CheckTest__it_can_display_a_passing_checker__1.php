<?php return 'Running checker: ProtoneMedia\\ApiHealth\\Tests\\TestCheckers\\PassingChecker
Passes!
';
