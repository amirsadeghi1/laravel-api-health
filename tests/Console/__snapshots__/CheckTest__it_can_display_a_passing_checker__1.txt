Running checker: ProtoneMedia\ApiHealth\Tests\TestCheckers\PassingChecker
Passes!
